CareEvolution/ResearchKit
# Add precision (by default) to Height/Weight Pickers

Closes https://github.com/CareEvolution/RKStudio-Participant/issues/1099. This is a simple approach to add precision mostly as discussed in https://github.com/CareEvolution/Consumers/issues/11101. 

ResearchKit currently does not have a high-precision concept for Height and for Weight and USC, it currently uses ounces.

While still allowing support for "low precision" (current behavior), the accompanying changes in [CEVResearchKit](https://github.com/CareEvolution/CEVResearchKit/pull/270) will now default to this new "high precision" mode.

|Mode|BEFORE|AFTER|
|---|---|---|
|**Height - USC**|![Height - USC - Low](https://user-images.githubusercontent.com/1008462/198047174-4e74302e-95b2-4fb5-9cd6-4563266b8543.png)|![Height - USC - High](https://user-images.githubusercontent.com/1008462/198047267-3bd45ff7-8d09-4b7c-8fb2-cceda3a2d302.png)|
|**Height - Metric**|![Height - Metric - Low](https://user-images.githubusercontent.com/1008462/198047465-21aabc8d-5af0-4ae4-862d-b44a4c3fadb1.png)|![Height - Metric - High](https://user-images.githubusercontent.com/1008462/198047512-0c07d1fb-59ea-4bc0-a495-d4a4cbb9e78f.png)|
|**Weight - USC**|![Weight - USC - Low](https://user-images.githubusercontent.com/1008462/198047620-7fc6d8c4-f7d3-495d-9e33-372e700d4091.png)|![Weight - USC - High](https://user-images.githubusercontent.com/1008462/198047651-f7aba0a8-dfb4-4b23-bfa4-52b6c0b30671.png)|
|**Weight - Metric**|![Weight-Metric-Low](https://user-images.githubusercontent.com/1008462/198047782-c4b52835-de6e-4d9c-b206-b4510a8c6984.png)|![Weight - Metric - High](https://user-images.githubusercontent.com/1008462/198047795-2ff8561a-d22b-4ff1-8da9-87b5e537ed2c.png)|

### Testing
- Check to see that for a **Question Step** or **Form Step** that uses an **Answer Format** of **Height**  shows ability to 
select fraction of inches as shown above
- Check to see that for a **Question Step** or **Form Step** that uses an **Answer Format** of **Weight**  shows ability to select fraction of pounds as shown above
